### DISCUSSION TITLE
Error in ftINIT (Failed to find good enough solution within the time frame. MIPGap: Inf)

### DISCUSSION THREAD

**Discussion by hazelweeling** [SysBioChalmers/Human-GEM#536] [Q&A]
Hi there, 

I've created an animal model using RAVEN's _getModelFromHomology_ using Human-GEM as a template. I'm currently trying to run ftINIT to create a contextualised model using RNA sequencing data. 

However, I'm running into an error (shown below). 
<img width="841" alt="Screenshot 2023-04-17 at 09 14 29" src="https://user-images.githubusercontent.com/92432310/232427210-fcd7d0c0-cc53-481c-b76d-f78d36722692.png">

Any help or advice would be appreciated. Thank you!

**Comment by rsango6** [comment]
Hi all,

I am stuck at the same step with the same error:

```
...
Model is infeasible
Best objective -, best bound -, gap -
Error using dispEM
Failed to find good enough solution within the time frame. MIPGap: Inf


Error in ftINIT (line 281)
        dispEM(['Failed to find good enough solution within the time frame. MIPGap: ' num2str(mipGap)]);

```

Would also appreciate any advice on this.

**Comment by mihai-sysbio** [comment]
Linking here two other threads:
- https://github.com/SysBioChalmers/Human-GEM/issues/511
- https://github.com/SysBioChalmers/RAVEN/issues/483
